SConstruct

```diff
@@ -506,13 +537,13 @@ config_options = [
         'blas_lapack_dir',
         """Directory containing the libraries specified by 'blas_lapack_libs'. Not
            needed if the libraries are installed in a standard location, for example,
-           ``/usr/lib``.""",
+           '/usr/lib'.""",
         '', PathVariable.PathAccept),
     EnumVariable(
         'lapack_names',
         """Set depending on whether the procedure names in the specified
            libraries are lowercase or uppercase. If you don't know, run 'nm' on
-           the library file (for example, 'nm libblas.a').""",
+           the library file (for example, "nm libblas.a").""",
```

Comment:
This would be OK to leave as `'` since it's representing a program/CLI thingy, and that's our convention elsewhere.

Reply:
That's unfortunately a little tricky, as it's very hard to guess what single quotes enclose (for example, you'd had to catch exceptions for all kinds of contractions "isn't", "doesn't", "can't", "you'd", "that's"). My way around this was to limit single quote extents to whatever doesn't include whitespace. So using double quotes here is imho the easier solution (and it is still clear from a CLI perspective).